src/targets/betafpv_2400.ini

```diff
@@ -34,6 +34,20 @@ lib_deps =
 [env:BETAFPV_2400_TX_MICRO_via_WIFI]
 extends = env:BETAFPV_2400_TX_MICRO_via_UART
 
+[env:BETAFPV_2400_TX_MICRO_1000mw_via_UART]
+extends = env:BETAFPV_2400_TX_MICRO_via_UART
+build_flags = 
+	${env_common_esp32.build_flags}
+	${common_env_data.build_flags_tx}
+	-include target/BETAFPV_2400_TX_MICRO.h
+	-D VTABLES_IN_FLASH=1
+	-O2
+	-D MICRO_1000MW
```

Comment:
Should this be `BETAFPV_MICRO_1000MW` since it is for a specific target?

Reply:
Yea, I was thinking about that. 

I thinking having it BETAFPV_MICRO_1000MW would be more specific but figured MICRO_1000MW would only be defined with BetaFPV stuff. 

I can change it to that.